src/plugins/mf_priority.cpp

```diff
@@ -108,6 +108,13 @@ int64_t priority_calculation (flux_plugin_t *p,
     fshare_factor = b->fairshare;
     queue_factor = b->queue_factor;
 
+    priority = round ((fshare_weight * fshare_factor) +
+                      (queue_weight * queue_factor) +
+                      (urgency - 16));
+
+    if (fshare_factor < 0.00000)
```

Comment:
Question: The commit message states:

>  The multi-factor priority plugin currently rejects jobs from a user who has a fairshare value of 0.0.

Is this supposed to be "rejects jobs from a user who has a fairshare value of _less than_ 0.0" ?

Also, could this test be moved above the calculation of `priority` since it immediately returns? (Thus the `priority` is done needlessly) Apologies if I'm misreading the code in diff format.